### PR TITLE
[Workaround] Disable conv_hip_implicit_gemm_grouped_bwd_xdlops

### PR DESCRIPTION
**Pull request by msaudulhassan**

### CODE REVIEW

File: src/solver/conv_hip_implicit_gemm_grouped_bwd_xdlops.cpp [hunk 1]
```diff
@@ -460,7 +460,7 @@ bool ConvHipImplicitGemmGroupBwdXdlops::IsApplicable(
     [[maybe_unused]] const ProblemDescription& problem) const
 {
 #if MIOPEN_BACKEND_HIP && MIOPEN_USE_COMPOSABLEKERNEL
-    if(miopen::IsDisabled(ENV(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_HIP_GROUP_BWD_XDLOPS)))
+    if(miopen::IsEnabled(ENV(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_HIP_GROUP_BWD_XDLOPS)))
```
Comment (junliume) [inline src/solver/conv_hip_implicit_gemm_grouped_bwd_xdlops.cpp:463]: @msaudulhassan interesting way to disable a solver ;-P I like it that we can test the fix later by enabling that env var explicitly.

Comment (msaudulhassan) [reply src/solver/conv_hip_implicit_gemm_grouped_bwd_xdlops.cpp:463]: @junliume PS: I'll set this PR to Open once I have tested it on the detectron2 workload

**Comment by junliume** [comment]
@zjing14 @JehandadKhan : I think we still need this WA until the kernel issues is fully investigated and fixed.

**Comment by msaudulhassan** [comment]
> @zjing14 @JehandadKhan : I think we still need this WA until the kernel issues is fully investigated and fixed.

Yes we'd still need it even if TunaNet is fixed. Unfortunately, I have been mainly been focusing on fixing Tunanet thus far and haven't been able to get to this. The PR fixing TunaNet should be open by the day's end and then I can resume this PR

**Review by junliume — APPROVED** [review]

**Pull request merged**